test/src/DataManagement/BundleDataManagementTests.js

```diff
@@ -0,0 +1,330 @@
+const mocha = require('mocha');
+const fs = require('fs-extra');
+const { expect } = require('chai');
+const path = require('path');
+
+const BundleDataManager = require(path.join(__dirname, '..', '..', '..', 'src', 'DataManagement', 'BundleDataManager'));
+
+describe("Bundle Data Tests", function() {
+  let dataManager = new BundleDataManager();
+
+  before(function() {
+    fs.mkdirSync(path.join(__dirname, 'test-bundle-data'));
+    fs.mkdirSync(path.join(__dirname, 'test-bundle-data', 'bundles'));
+    dataManager.loadBundles(path.join(__dirname, 'test-bundle-data'));
+    dataManager
+      .createBundle('rooms-bundle')
+      .createArea({
+        name: 'new-area',
+        title: 'New Area',
+        suggestedLevel: '1-10'
+      });
+  });
+
+  after(function() {
+    dataManager.deleteBundle('rooms-bundle');
+    dataManager.deleteBundle('awesome-bundle');
+    fs.removeSync(path.join(__dirname, 'test-bundle-data'));
+  });
+
+  describe("BundleDataManagement", function() {
+    it("loadBundles loads bundles", function() {
+      expect(dataManager.bundles['rooms-bundle']).to.not.be.null;
+    });
+
+    describe("bundleExists", function() {
+      it("returns true when bundle exists", function() {
+        expect(dataManager.bundleExists('rooms-bundle')).to.be.true;
+      });
+
+      it("returns false when bundle doesn't exist", function() {
+        expect(dataManager.bundleExists('not-a-real-bundle')).to.be.false;
+      });
+    });
+
+    describe("getBundleDirectory", function() {
+      it("getBundleDirectory returns a bundle if it exists", function() {
+        expect(dataManager.getBundleDirectory('rooms-bundle')).to.not.be.null;
```

Comment:
A better test could be `expect(...).to.be.ok` -- more expressive, and makes sure it is neither null nor undefined.